examples/applications/restapi/tests/test_restapi_handlers.py

```diff
@@ -0,0 +1,549 @@
+import asyncio
+import json
+
+import bitcoinx
+from aiohttp import web
+from electrumsv.restapi import good_response, Fault
+from aiohttp.test_utils import make_mocked_request
+from concurrent.futures.thread import ThreadPoolExecutor
+import pytest
+from bitcoinx import Address, Script, BitcoinTestnet
+import logging
+from typing import List, Union, Dict, Any, Optional, Tuple
+import tempfile
+from electrumsv.wallet import ParentWallet, Abstract_Wallet, UTXO
+from electrumsv.transaction import Transaction
+from ..handlers import ExtensionEndpoints
```

Comment:
Imports are not structured correctly. Should be standard library, then third party packages, then electrumsv, then electrumsv gui imports.